README.md

```diff
@@ -4,23 +4,52 @@ Ramsey Solutions Frontend configurations and standards.
 
 ## Packages
 
+Packages are published to NPM within the [`ramseyinhouse`](https://www.npmjs.com/org/ramseyinhouse) organization.
```

Comment:
I think the appropriate way to refer to it is `npm`, even in written copy. It's this way in their [own copy](https://www.npmjs.com/get-npm), as well as external sources like [Wikipedia](https://en.wikipedia.org/wiki/Npm_(software)).

```suggestion
Packages are published to npm within the [`ramseyinhouse`](https://www.npmjs.com/org/ramseyinhouse) organization.
```